ChangeLog

```diff
@@ -8,6 +8,13 @@ What's New in astroid 2.5.0?
 Release Date: TBA
 
 
+What's New in astroid 2.4.2?
```

Comment:
Please remove this, 2.4.1 has not been released yet.

Reply:
Ah whoops. I'll move it to 2.4.1